src/IO/MultiscaleSpatialImage.js

```diff
@@ -5,6 +5,9 @@ import componentTypeToTypedArray from './componentTypeToTypedArray'
 import WebworkerPromise from 'webworker-promise'
 import ImageDataFromChunksWorker from './ImageDataFromChunks.worker'
 import { CXYZT, toDimensionArray } from './dimensionUtils'
+
+export const MAX_COMPONENT_COUNT = 4
```

Comment:
We have the limitation in vtk.js rendering for a max component count from WebGL, but this is imposed with the number of selected components that are enabled on the input image. Those selected components are used to generate the fusedImage, which results in an in-memory vtkImageData with a potential subset of components. Can we keep that functionality? That is, ConglomerateMultiscaleSpatialImage can have N components. A MAX_COMPONENT_COUNT is a limitation of the vtk.js renderer, and the selected component subset is defined in the rendering and rendering options.

Reply:
OK, I'll fix.

Reply:
:pray: